test/fidelity.js

```diff
@@ -26,6 +31,21 @@ describe("Tests to ensure that idiomorph merges properly", function(){
         testFidelity("<button class=\"foo\">Foo</button>", "<button class=\"bar\">Foo</button>")
     });
 
+    it('morphs mutliple attributes correctly and back', function ()
```

Comment:
```suggestion
    it('morphs multiple attributes correctly and back', function ()
```
Typo in the description